pallets/gear-program/src/pause.rs

```diff
@@ -110,7 +110,12 @@ impl<T: Config> pallet::Pallet<T> {
 
         PausedPrograms::<T>::remove(program_id);
 
-        common::set_program_and_pages_data(program_id, paused_program.program, memory_pages);
+        if let Err(err) =
+            common::set_program_and_pages_data(program_id, paused_program.program, memory_pages)
+        {
+            log::error!("Wrong page with data number: {:?}", err.0);
```

Comment:
It's better to implement `fmt::Display` for error

Reply:
ok